docs/releases/5.2.txt

```diff
@@ -334,6 +334,15 @@ Models
   that the expression allows composite expressions, for example, to support
   :ref:`composite primary keys <cpk-and-database-functions>`.
 
+* The new ``order_by`` argument for :class:`~django.db.models.Aggregate` allows
+  to specify the ordering of the elements in the result.
+
+* The new :attr:`.Aggregate.allow_order_by` class attribute determines whether
+  the aggregate function allows passing an ``order_by`` keyword argument.
+
+* The new :class:`~django.db.models.StringAgg` aggregate returns the input
+  values concatenated into a string, separated by the ``delimiter`` string.
```

Comment:
I wonder if we should mention that this aggregate used to only be supported on Postgres